frontend/src/components/LevelSelectionBox/LevelSelectionBox.css

```diff
@@ -22,6 +21,7 @@
 
 .level-button.selected {
 	background-color: var(--level-button-background-colour-selected);
+	color: var(--level-button-colour);
```

Comment:
Actually, I wonder if we could put `cursor: default` here on the selected level, instead of using pointer, to make it clearer that it's not actionable when the level is already selected.